client/src/main/kotlin/com/classpass/moderntreasury/fake/ModernTreasuryFake.kt

```diff
@@ -97,19 +97,24 @@ open class ModernTreasuryFake :
     override fun getLedgerAccounts(
         ledgerAccountIds: List<LedgerAccountId>,
         balancesAsOfDate: LocalDate?,
-        page: Int,
+        afterCursor: LedgerAccountId?,
         perPage: Int
     ): CompletableFuture<ModernTreasuryPage<LedgerAccount>> = supplyAsync {
         // Create a local copy of ledger account info with computed balance. Does not mutate shared.
         val accounts = ledgerAccountIds.mapNotNull {
             accounts[it]?.copy(balances = getBalances(it, asOfDate = balancesAsOfDate))
         }
+
+        val accountsAfterCursor = accounts.takeLastWhile { afterCursor != it.id }
+
+        val content = accountsAfterCursor.take(perPage)
+        val nextAfterCursor =
+            if (accountsAfterCursor.size <= perPage) null else content.lastOrNull()?.id.toString()
```

Comment:
This is more like how it works:
```suggestion
            if (accountsAfterCursor.size == 0) null else content.lastOrNull()?.id.toString()
```
We want to allow for alternative implementations to sometimes return fewer results on a page.

Reply:
Yup - that makes sense